src/status_im2/contexts/chat/messages/composer/controls/view.cljs

```diff
@@ -40,12 +40,12 @@
      :size 32} :i/reaction]])
 
 (defn image-button
-  []
+  [bottom-inset]
   [quo/button
    {:on-press (fn []
                 (permissions/request-permissions
                  {:permissions [:read-external-storage :write-external-storage]
-                  :on-allowed  #(rf/dispatch [:open-modal :photo-selector])
+                  :on-allowed  #(rf/dispatch [:open-modal :photo-selector {:bottom-inset bottom-inset}])
```

Comment:
Hi @OmarBasem, why we are making these changes in PR
1. Changes doesn't look related to issue itself
2. It looks very confusing variable name is also bottom-inset, it also has a key with same name

Reply:
@Parveshdhull this is reverting some changes from that PR https://github.com/status-im/status-mobile/pull/15467

For now, we need to use the insets from screen-params. Not doing so causes the photo selector on Android to be unusable (the bottom gradient cover the whole screen due to incorrect insets).

There is an open issue to change the way we use insets: https://github.com/status-im/status-mobile/pull/15467#discussion_r1153727719